packages/js/src/executors/tsc/tsc.impl.ts

```diff
@@ -60,6 +61,8 @@ export async function* tscExecutor(
     options.tsConfig = tmpTsConfig;
   }
 
+  addTslibDependencyIfNeeded(options, context, dependencies);
```

Comment:
I think this is fine for the time being, eventually a lot of the js specific logic should be pulled out to use `processProjectGraph` like a community plugin would, to truly separate the core and at that point then some of this may make more sense there. Since this is specific to the `tsc` executor (and **_not_** wanted for the `swc` executor), I think this makes sense.

Reply:
Yeah, I figured there's something missing to do what I had in mind. Thanks for the feedback 👍🏻 I'll push a fix momentarily